### PR TITLE
Update image storage

### PR DESCRIPTION
**Pull request by chihsuan**
Issue: https://github.com/Canner/canner/issues/97

Update image and gallery component with the property imageStorage.

### CODE REVIEW

File: packages/antd-object-image/src/index.js [hunk 1]
```diff
@@ -82,7 +81,7 @@ export default class Image extends PureComponent<Props, State> {
         <EditImage
           onChange={this.onChange}
           editPopup={editPopup}
-          serviceConfig={imageServiceConfig}
+          imageStorage={imageStorage}
```
Comment (abz53378) [inline packages/antd-object-image/src/index.js]: @chihsuan, I prefer to pass a `serviceConfig` here
```js
serviceConfig={{
  customRequest: ({..}) => {...}
}}
```

File: docs/demos/array/gallery.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ export default class GalleryDemo extends React.Component<ArrayTypes<any>> {
           <Gallery
             value={value}
             refId={new RefId("gallery")}
-            imageServiceConfig={new ImgurService({
+            imageStorage={new ImgurStorage({
```
Comment (abz53378) [inline docs/demos/array/gallery.js:45]: @chihsuan, I prefer to pass a `serviceConfig` here
```js
serviceConfig={{
  customRequest: ({..}) => {...}
}}
```

**Review by abz53378 — CHANGES_REQUESTED** [review]

File: docs/demos/array/gallery.js (hunk 2 above)
Comment (chihsuan) [reply docs/demos/array/gallery.js:45]: @abz53378: I've updated the components. But should this be like the image component that is written in the `packages/antd-array-gallery/src/index.js` or you prefer to pass serviceConfig here?

Comment (abz53378) [reply docs/demos/array/gallery.js:45]: Yes, you are right. I comment at the wrong file. 😭

**Review by abz53378 — APPROVED** [review]

**Pull request merged**